cudamapper/src/application_parameters.cpp

```diff
@@ -219,6 +222,27 @@ void ApplicationParameters::create_input_parsers(std::shared_ptr<io::FastaParser
         target_parser = io::create_kseq_fasta_parser(target_filepath, kmer_size + windows_size - 1);
     }
 
+    number_of_basepairs_t total_sequence_length                 = 0;
+    const number_of_basepairs_t minimum_for_automatic_filtering = 500000; // Require at least 0.5Mbp of sequence for filtering by default
+    number_of_reads_t query_index                               = 0;
+    number_of_reads_t target_index                              = 0;
```

Comment:
optional: this can be only one counter and just set back to zero before going for target sequences, they are never used simultaneously

Reply:
Is that the preferred style? I had initially done this, but introduced the bug where I didn't reset the counter...

Reply:
Makes sense, let's leave it as it is